tests/test_get_avatar.py

```diff
@@ -1,56 +1,56 @@
-import unittest
-import json
+import os.path
+
+import pytest
 import requests as r
 import yaml
-import pytest
 from parameterized import parameterized
-import logging
+
 import username_api
-import os.path
 
 data = yaml.load(open(os.path.join('tests', 'test_data.yml')))
 websites = yaml.load(open('websites.yml'))
 
 
-def load_test_cases(type):
-	res = []
-	for website in data:
-		if website in ['behance', 'openhub']:
-			users = [None]
-		elif type == 'with_avatar':
-			if 'avatar_usernames' in data[website]:
-				users = data[website]['avatar_usernames']
-			else:
-				users = data[website]['taken_usernames']
-		else:
-			users = data[website]['available_usernames']
-		res.extend((website, user) for user in users)
+def load_test_cases(type_):
+    res = []
+    for website in data:
+        if website in ['behance', 'openhub']:
+            users = [None]
+        elif type_ == 'with_avatar':
+            if 'avatar_usernames' in data[website]:
+                users = data[website]['avatar_usernames']
+            else:
+                users = data[website]['taken_usernames']
+        else:
+            users = data[website]['available_usernames']
+        res.extend((website, user) for user in users)
 
-	return res
+    return res
 
 
 def custom_name_func(testcase_func, param_num, param):
-	return '%s_%s' % (
-		testcase_func.__name__,
-		parameterized.to_safe_name('_'.join(str(x) for x in param.args)),
-	)
-
-
-class TestGet_avatar(object):
-
-	@parameterized.expand(load_test_cases('with_avatar'),
-						  testcase_func_name=custom_name_func)
-	def test_with_avatar(self, website, user):
-		if not user:
-			pytest.skip("website not supported")
-		link = username_api.check_username(website, user)['avatar']
-		response = r.get(link)
-		assert(response.headers.get('content-type', '').startswith('image/') or
-			   response.headers.get('content-type') == 'application/octet-stream')
-
-	@parameterized.expand(load_test_cases('without_avatar'),
-						  testcase_func_name=custom_name_func)
-	def test_without_avatar(self, website, user):
-		if not user:
-			pytest.skip("website not supported")
-		assert username_api.check_username(website, user)['avatar'] is None
+    return '%s_%s' % (
+        testcase_func.__name__,
+        parameterized.to_safe_name('_'.join(str(x) for x in param.args)),
+    )
+
+
+class TestGetAvatar(object):
+
+    @parameterized.expand(load_test_cases('with_avatar'),
+                          testcase_func_name=custom_name_func)
+    def test_with_avatar(self, website, user):
+        if not user:
+            pytest.skip('website not supported')
+        link = username_api.check_username(website, user)['avatar']
+        response = r.get(link)
+        assert(response.headers.get('content-type', '').startswith('image/') or
```

Comment:
Unrelated to your patch, but good to include now as it is another whitespace only change ...

`assert(..` looks like a function call.

instead, adding a space can help ensure people see a function call was not intended

`assert (..`